codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/protocol/ServerProtocolTestGenerator.kt

```diff
@@ -263,11 +275,11 @@ class ServerProtocolTestGenerator(
             );
             """
         )
-        if (testCase.body != null) {
+        if (!testCase.body.isEmpty()) {
             rustTemplate(
                 """
                 let body = #{Hyper}::body::to_bytes(http_response.into_body()).await.expect("unable to extract body to bytes");
-                assert_eq!("${testCase.body.get().replace("\"", "\\\"")}", body);
+                assert_eq!(${testCase.body.get().dq().replace("#", "##")}, body);
```

Comment:
you should use `escape(testCase.body.get()).dq()`

Reply:
Ah, this is actually what I wanted to do. I really should spend some time in checking all the helpers that already exists in the utils..